partcad/src/partcad/part_factory_obj.py

```
@@ -0,0 +1,152 @@
+                for vertex_idx in face:
+                    x, y, z = vertices[vertex_idx - 1]
+                    polygon.Add(gp_Pnt(x, y, z))
+                polygon.Close()
+                shape_faces.append(BRepBuilderAPI_MakeFace(polygon.Wire()).Face())
+
+            return shape_faces  # Optionally combine faces into a compound
+        except Exception as e:
+            pc_logging.error(f"Error loading OBJ file: {e}")
+            raise
+
+    async def _process_obj_subprocess(self):
```

Ruff (0.8.2)

Missing return type annotation for private function `_process_obj_subprocess`

(ANN202)